default_no_bzr.cfg

```diff
@@ -0,0 +1,5 @@
+[buildout]
```

Comment:
@nbessi This was done in an attempt to accelerate the builds. Hum, guess there is no use for this file now, I'm closing this PR.